conan/internal/cache/db/recipes_table.py

```diff
@@ -26,6 +26,15 @@ def _as_dict(row):
             "lru": row.lru,
         }
 
+    @staticmethod
+    def _as_dict_no_rev_timestamp(row):
```

Comment:
Maybe not worth to make it its own method and better to inline the function in `all_references`?

Reply:
Yes, sounds it would be better.